src/commands/node/get.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { flags as flagParser } from '@oclif/command';
+import BaseCommand from '../../base';
+import getAPIClient from '../../utils/api';
+
+export default class GetCommand extends BaseCommand {
+	async run() {
+		const { flags: { all } } = this.parse(GetCommand);
+		const client = getAPIClient(this.userConfig.api);
+		const baseInfo = await Promise.all([
+			client.node.getConstants(),
+			client.node.getStatus(),
+		]).then(([constantsResponse, statusResponse]) => ({
+			...constantsResponse.data,
+			...statusResponse.data,
+		}));
+		if (!all) {
+			return this.print(baseInfo);
+		}
+		try {
+			const forgingResponse = await client.node.getForgingStatus();
+			return this.print({
+				...baseInfo,
+				forgingStatus: forgingResponse.data || [],
+			});
+		} catch (error) {
+			return this.print({
+				...baseInfo,
+				forgingStatus: error.message,
+			});
+		}
+	}
+}
+
+GetCommand.flags = {
+	...BaseCommand.flags,
+	all: flagParser.boolean({
```

Comment:
`all` sounds to me like you might have a set of nodes and this is going to get information on all of them. How about `--forging-status` or `full`?

Reply:
yea that's true. `--forging-status` sounds good to me